notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.                "ignored": {
</pre><p>How do you know which layers should be ignored? It would be good to have the explanation why them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>added</p>